x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/indexlifecycle/TimeseriesLifecycleType.java

```diff
@@ -72,6 +68,13 @@ public String getWriteableName() {
         for (String phaseName : VALID_PHASES) {
             Phase phase = phases.get(phaseName);
             if (phase != null) {
+                Map<String, LifecycleAction> actions = phase.getActions();
```

Comment:
Can you add a unit test for this? Hopefully you should be able to instantiate a `TimeseriesLifecycleType` and directly call this since it's public.

Reply:
Done, thre is now a unit test for this.